src/common/OptionsMenu/OptionsMenu.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import PopUpDialog from '../PopUpDialog/PopUpDialog'
+
+import { CSSTransition } from 'react-transition-group'
+import './optionsMenu.scss'
+
+const OptionsMenu = React.forwardRef(({ children, show, timeout }, ref) => {
+  return (
+    <CSSTransition
+      in={show}
+      timeout={timeout}
+      classNames="options-menu"
+      unmountOnExit
+    >
+      <PopUpDialog
+        className="select__options-list"
+        customPosition={{
+          element: ref,
+          position: 'bottom-right'
+        }}
+        style={{ width: 'auto' }}
```

Comment:
We should avoid using `style` attribute if it's possible. In the `Select` component we have dynamic variable, that's why it was used there.
Also this style is redundant. You can delete it. If you inspect the page you will see that this element has already the property `width: auto`